docs/sdks/server-sdks/dotnet.md

```diff
@@ -49,60 +75,262 @@ internal class AssignmentLogger : IAssignmentLogger
+| `Subject` (string)                        | An identifier of the subject or user assigned to the experiment variation                                                | UUID                                |
+| `Variation` (string)                      | The experiment variation the subject was assigned to                                                                     | "control"                           |
+| `Timestamp` (DateTime)                    | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z            |
+| `SubjectAttributes` (Map<String, object>) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `Map.of("device","iOS")`            |
+| `FeatureFlag` (string)                    | An Eppo feature flag key                                                                                                 | "recommendation-algo"               |
+| `Allocation` (string)                     | An Eppo allocation key                                                                                                   | "allocation-17"                     |
+
+
+## Usage with Contextual Multi-Armed Bandits
+
+Eppo also supports contextual multi-armed bandits. You can read more about them in the [high-level documentation](../../../contextual-bandits).
+Bandit flag configuration--including setting up the flag key, status quo variation, bandit variation, and targeting rules--are configured within
+the Eppo application. However, available actions are supplied to the SDK in the code when querying the bandit.
+
+To leverage bandits using the Node SDK, there are two additional steps over regular feature flags:
+1. Add a bandit action logger to the SDK client instance
+2. Query the bandit for an action
+
+
+### Defining a Bandit Logger
+
+In order for the bandit to learn an optimized policy, we need to capture and log the bandit's actions.
+This requires defining a bandit logger in addition to an assignment logger.
+
+```cs
+class SegmentLogger : IAssignmentLogger
+{
+    private readonly Analytics analytics;
+
+    public SegmentLogger(Analytics analytics)
+    {
+        this.analytics = analytics;
+    }
+
+    public void LogAssignment(AssignmentLogData assignmentLogData)
+    {
+        analytics.Track("Eppo Randomization Assignment", assignmentLogData);
+    }
 
+    public void LogBanditAction(BanditLogEvent banditLogEvent)
+    {
+        analytics.Track("Eppo Bandit Action", banditLogEvent);
+    }
+}
 ```
-eppoClient.GetBoolAssignment(string subjectKey, string flagKey, SubjectAttributes? subjectAttributes = null)
-eppoClient.GetNumericAssignment(string subjectKey, string flagKey, SubjectAttributes? subjectAttributes = null)
+
+The SDK will invoke the `logBanditAction()` function with a `BanditLogEvent` object that contains the following fields:
```

Comment:
I believe we want `LogBanditAction()` to be capitalized (silly dot net conventions 🙄 )